lib/reactive_table.js

```diff
@@ -295,15 +296,21 @@ var getDefaultFieldVisibility = function (field) {
     return new ReactiveVar(!field.hidden || (_.isFunction(field.hidden) && !field.hidden()));
 }
 
+var getRowCount = function () {
+    if (this.server) {
+        return ReactiveTableCounts.findOne(this.publicationId.get());
```

Comment:
this could do the (count ? count.count : 0) check and getPageCount wouldn't have to distinguish the server-side collections